ansible/docker/Jenkinsfile.test

```diff
@@ -0,0 +1,132 @@
+pipeline {
+    agent none
+    stages {
+        stage('Docker Build') {
+            parallel {
+                stage('Ubuntu24.04 x64') {
+                    agent {
+                        label "dockerBuild&&linux&&x64&&dockerhost-azure-ubuntu2204-x64-1"
+                    } 
+                    steps {
+                        dockerBuild('amd64', 'ubuntu2404', 'Dockerfile.u2404')
+                    }
+                }
+                stage('Ubuntu24.04 aarch64') {
+                    agent {
+                        label "dockerBuild&&linux&&aarch64"
+                    } 
+                    steps {
+                        dockerBuild('arm64', 'ubuntu2404', 'Dockerfile.u2404')
+                    }
+                }
+                stage('UBI10 x64') {
+                    agent {
+                        label "dockerBuild&&linux&&x64&&dockerhost-azure-ubuntu2204-x64-1"
```

Comment:
Should this be pinned to a named host?.. I know we probably only have that one capable one :)

Reply:
That's a fair comment (both parts of it! We used to have a skytap x64 dockerhost at one point too...) Those tags without the hostname would also trigger on the dynamic azure machines too so I may have stuck that one a while back to prevent that for debugging purposes. I'll give it a run through without that fixed hostname.